pkg/cache/cache.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2017 Envoyproxy Authors
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+// Package cache defines a configuration cache for the server
+package cache
+
+import (
+	"github.com/envoyproxy/go-control-plane/api"
+	"github.com/golang/protobuf/proto"
+)
+
+// ConfigWatcher requests watches for configuration resources by a node, last
```

Comment:
One space after the period please, here and throughout. http://www.quickanddirtytips.com/education/grammar/two-spaces-after-a-period

Reply:
This is my vim adding spaces. Edited.